Makefile

```diff
@@ -166,7 +166,6 @@ govet: $(find . -type f -name '*.go' -not -path "./vendor/*" -not -path "apiserv
 	@echo checking go vet...
 	@$(GO) tool vet -all $$(find . -type f -name '*.go' -not -path "./vendor/*")
 	@$(GO) tool vet -shadow $$(find . -type f -name '*.go' -not -path "./vendor/*")
-	@touch $@
```

Comment:
wondered why that was there.  suppose you could revert b2c4261 with this gone, but no biggie.